Library/Homebrew/test/release_notes_spec.rb

```diff
@@ -12,21 +12,27 @@
       system "git", "commit", "--allow-empty", "-m", "Merge pull request #1 from Homebrew/fix", "-m", "Do something"
       system "git", "commit", "--allow-empty", "-m", "make a change"
       system "git", "commit", "--allow-empty", "-m", "Merge pull request #2 from User/fix", "-m", "Do something else"
+      system "git", "commit", "--allow-empty", "-m", "another change"
+      system "git", "commit", "--allow-empty", "-m", "Merge pull request #3 from User/another_change"
     end
   end
 
   describe ".generate_release_notes" do
     it "generates release notes" do
+      # rubocop:disable Style/TrailingWhitespace
       expect(described_class.generate_release_notes("release-notes-testing", "HEAD")).to eq <<~NOTES
+        https://github.com/Homebrew/brew/pull/3 (@User) - 
         https://github.com/Homebrew/brew/pull/2 (@User) - Do something else
         https://github.com/Homebrew/brew/pull/1 (@Homebrew) - Do something
       NOTES
+      # rubocop:enable Style/TrailingWhitespace
     end
 
     it "generates markdown release notes" do
       expect(described_class.generate_release_notes("release-notes-testing", "HEAD", markdown: true)).to eq <<~NOTES
-        - [Do something else](https://github.com/Homebrew/brew/pull/2) (@User)
-        - [Do something](https://github.com/Homebrew/brew/pull/1) (@Homebrew)
+        - [ #3](https://github.com/Homebrew/brew/pull/3) (@User)
```

Comment:
Similarly this output seems a bit weird. How about `Merge pull request #3 from User/another_change` or anything relating to `another_change` (as it's a branch name)?

Reply:
I've used the subject line of the commit if the body is empty (so `Merge pull request #3 from User/another_change`)